tests.py

```diff
@@ -368,3 +386,116 @@ def test_version(self, testbot):
         assert '1.0.0' in response
 
 
+@pytest.fixture(scope="module")
+def github_api():
+    return GithubAPI()
+
+
+def test_github_fetch_repos(github_api, mocker):
+    github_mock = mocker.patch.object(github_api, '_github', autospec=True)
+    repos = ['conda-devenv', 'deps', 'alfasim-sdk']
+    github_mock.get_organization.return_value.get_repos.return_value = repos
+
+    assert github_api.fetch_repos("esss") == repos
+    github_mock.get_organization.assert_called_with('esss')
+
+
+def test_github_fetch_branches(github_api, mocker):
+    github_mock = mocker.patch.object(github_api, '_github', autospec=True)
+    github_mock.get_organization.return_value.get_repo.return_value.get_branch.return_value = \
+        'single-branch'
+
+    github_mock.get_organization.return_value.get_repo.return_value.get_branches.return_value = \
+        ['branch-1', 'branch-2', 'branch-3']
+
+    branches = github_api.fetch_branches('esss', 'alfasim-sdk', branch_name='something')
+    assert branches == ['single-branch']
+
+    branches = github_api.fetch_branches('esss', 'alfasim-sdk')
+    assert branches == ['branch-1', 'branch-2', 'branch-3']
+
+
+def test_github_delete_branch(github_api, mocker):
+    github_mock = mocker.patch.object(github_api, '_github', autospec=True)
+    git_repo = GitRepo('repo')
+    github_mock.get_organization.return_value.get_repo.return_value = git_repo
+
+    with pytest.raises(AssertionError, match="Trying to delete the wrong branch, check the PR ID."):
```

Comment:
Yes Martin. GithubAPI is the name of Guilherme's github class.
Maybe rename `GithubAPI` to `Github` and `StashAPI` to `Stash`?

Reply:
Renaming to `Github` and `Stash` is not a good idea because that's the name of the libs we're using. Maybe in the future we should rename those to `GithubFacade` and `BitbucketFacade`.